dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/version/VersionSet.java

```diff
@@ -43,13 +43,16 @@ public class VersionSet extends CamelCommand {
     @CommandLine.Option(names = { "--reset" }, description = "Reset by removing any custom version settings")
     boolean reset;
 
+    @CommandLine.Option(names = { "--local" }, description = "Retrieve configurations from current directory")
```

Comment:
"Store ... to" over "Retrieve ... from"

Reply:
Or if we want the same wording for all options, we should consider a more generic wording. Examples:
"Apply configrations to current directoyr", or
"Use local configurations instead of global"?